protocol/requests/create_account.go

```diff
@@ -80,6 +80,7 @@ type CreateAccount struct {
 	APIConfig *APIConfig `json:"apiConfig"`
 
 	KeycardInstanceUID     string  `json:"keycardInstanceUID"`
+	KeycardPairingKey      string  `json:"keycardPairingKey"`
```

Comment:
Can you please mention here that only `KeycardPairingKey` can be used  instead and has precedence over `KeycardPairingKey`?